src/Components/Fields/CurrentLastName.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import FieldContainer from 'Components/FieldContainer';
+import {getField} from "Utils/fieldParser";
+
+function CurrentLastName(props){
+    const uuid = "b306238a-a0f6-4bb8-b8ea-b3216ca75e0b";
+    const field = getField(props.fieldContent, uuid);
+    const stateField = getField(props.stateData.nvrf_fields, field.uuid);
+
+    return (
+        <FieldContainer
+            fieldType={'text'} inputData={{
+            id: 'last_name',
+            dataTest: 'LastName',
```

Comment:
The test failed here - change to 'lastName' to make it match the test.

Reply:
ah ok, will do!